Example/Example/CharacteristicsController.swift

```diff
@@ -49,16 +51,28 @@ class CharacteristicsController: UIViewController {
             }).addDisposableTo(disposeBag)
     }
 
-    private func showWriteFieldForCharacteristic(characteristic: Characteristic) {
+    fileprivate func showWriteFieldForCharacteristic(characteristic: Characteristic) {
         let valueWriteController = UIAlertController(title: "Write value", message: "Specify value in HEX to write ",
                                                      preferredStyle: .alert)
         valueWriteController.addTextField { textField in
-
+            textField.delegate = self
```

Comment:
We're setting the delegate right here. But `UIAlertController` has property `textFields` which can give us the necessary data without introducing `Data` property in class.